src/index.js

```diff
@@ -121,7 +121,7 @@ const mapToPropertyDescriptor = (
 
       const attributeValue = (() => {
         if (typeOrSerDes === Types.json) {
-          return JSON.stringify(value);
+          return typeof value === 'string' ? value : JSON.stringify(value);
```

Comment:
This is property access here ... if you want to assign JSON, just use a regular JS Object or Array!